hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -48,38 +50,44 @@
 (def pid-file (str dir "/server.pid"))
 (def log-file (str dir "/server.log"))
 
+(def CLIENT_ID_SEPARATOR "xxxxxxxxxx")
```

Comment:
This feels a bit out of place for Clojure constants; consider (def client-id-separator ...)